content/release-notes/2023-02-14.md

```diff
@@ -5,7 +5,5 @@ label: 'Storage'
 ### Features
 
 - Compute: Added support for the the PostgreSQL `pgvector`, `plls` and `plcoffee` extensions. For more information about PostgreSQL extensions supported by Neon, see [PostgreSQL extensions](https://neon.tech/docs/reference/pg-extensions/).
-- Proxy: Latencies have been drastically decreased for websocket and pool connections.
-- Pageserver: Experimental layer eviction feature to reduce non used storage layers.
-- Pageserver: Documented the synthetic size calculation.
-
+- Proxy: Reduced network latencies for WebSocket and pooled connections to the Neon Proxy by enabling the `TCP_NODELAY` option. Enabling this option causes segments to be sent as soon as possible, even if there is only a small amount of data. For more information about the `TCP_NODELAY` option, refer to the [TCP protocol man page](https://linux.die.net/man/7/tcp).
```

Comment:
It's not only one item. @funbringer introduced the cache for this

Reply:
Hi @vadim2404 Is there a PR for that I can look it? The only thing regarding caching I see is this one, but it does not seem to be related: https://github.com/neondatabase/neon/pull/3605

Reply:
You mentioned valid PR. but there was one more https://github.com/neondatabase/neon/pull/3331

Reply:
Revised the entry to mention #3331